integration-tests/spring-boot-properties/README.md

```diff
@@ -0,0 +1,12 @@
+### Current differences with Spring Boot properties.
+
+1. Default values assigned with `=` operator are not supported.
+2. Getters and setters are needed for all fields, including nested classes.
+3. `@Bean` needs to be removed from a method which is annotated with `@ConfigurationProperties`.
+Otherwise two beans will be created and injection will fail.
+4. `@NestedConfigurationProperty` can be used but is not required.
+Nested property classes work out of the box with inner and outer classes.
+5. `@ConstructorBinding` is not supported.
+6. Spring Boot properties conversion is not supported.
+7. Random values are not supported
```

Comment:
What are random values?

Reply:
Spring Boot properties support various expressions in `application.properties`. One of them are random values where you can set something like this: `my.number=${random.int}`. Of course, if it's needed it should be a feature of Quarkus properties rather than Spring Boot properties extension

Reply:
I see, thanks!